README.md

```diff
@@ -50,7 +50,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://dev-mamun.github.io/microverse-portfolio/)
+- [Live Demo Link](https://dev-mamun.github.io/softexpo/)
```

Comment:
- [ ] I just noticed that after making changes you put the wrong live demo link in your `README` file and when clicked on the live demo link you provide we get this as the result :point_down: 
![Screenshot from 2023-02-16 18-17-19](https://user-images.githubusercontent.com/88894525/219425385-9459d2c3-741f-46ca-940b-38af2498df1b.png)

_[This the link we can see:](https://dev-mamun.github.io/microverse-portfolio/)